.github/workflows/test.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           --name=test_container \
           prairielearn/prairielearn /bin/bash
       - name: Run the JavaScript tests
-        run: docker exec --env GITHUB_ACTIONS=true test_container sh -c "make -s -C /PrairieLearn test-js"
+        run: docker exec --env GITHUB_ACTIONS=true test_container sh -c "make -s -C /PrairieLearn test-prairielearn-docker-smoke-tests"
```

Comment:
Since I can't comment on the line: should we still be uploading coverage reports from this job?

Reply:
Removed in [5a1ec66](https://github.com/PrairieLearn/PrairieLearn/pull/12153/commits/5a1ec66bc20279cb6b9a23ce5b31fdbb3ba95f46)